src/tpt_reports/tpt_reports.py

```diff
@@ -1,20 +1,21 @@
 """cisagov/tpt-reports: A tool for creating Technical Phishing Test (TPT) reports.
 
 Usage:
-  tpt-reports ASSESSMENT_ID ELECTION_NAME DOMAIN_TESTED JSON_FILE_PATH OUTPUT_DIRECTORY [--log-level=LEVEL]
+  tpt-reports ASSESSMENT_ID ELECTION_NAME DOMAIN_TESTED JSON_FILE_PATH [--output-dir=OUTPUT_DIRECTORY] [--log-level=LEVEL]
 
 Options:
   -h --help                         Show this message.
   -l --log-level=LEVEL              If specified, then the log level will be set to
                                     the specified value.  Valid values are "debug", "info",
                                     "warning", "error", and "critical". [default: info]
+  --output-dir=OUTPUT_DIRECTORY     The directory where the final PDF reports
```

Comment:
Is there a reason why you didn't include a short flag here, e.g. `-o` or `-d`?  Note that if you go with `-d`, you'll want to update the options to keep them in alphabetical order by short flag.

Reply:
I had the "-o" flag there initially but decided I would get opinions from the team first and forgot to add it back in. I think it makes sense to add "-o" for consistency here. I'll get it updated. Thanks!

Reply:
Resolved in a610b15

Reply:
Looks great! 👍